tasks.py

```diff
@@ -31,6 +43,40 @@
     HAS_RICH = False
 
 
+_ANSI_ESCAPE_RE = re.compile(r"\x1b\[[0-?]*[ -/]*[@-~]")
+_ROOT_PATH = Path(__file__).parent.absolute().resolve()
+_TESTS_TIMES_PATH = _ROOT_PATH / "tests-times.json"
+_TEST_RESULTS_DIR = ".test-results"
+
+_NAMES_DISTRIBUTION = [
+    "circuits",
+    "core",
+    "dcim",
+    "dcim/tests/test_api.py",
+    "dcim/tests/test_views.py",
+    "extras",
+    "extras/tests/test_api.py",
+    "extras/tests/test_jobs.py",
+    "extras/tests/test_views.py",
+    "ipam",
+    "tenancy",
+    "users",
+    "utilities",
+    "virtualization",
+]
+
+_SKIP_TESTS_NAMES = [
+    "/example_jobs/",
+    "/integration/",
+    "nautobot/core/tests/test_cli.py",
```

Comment:
Why's this test excluded?

Reply:
Explained

Reply:
Yes, these are excluded in `invoke unittest` by Django discovery (due to tags), and we just mimic this behavior by listing patterns to skip. Not the best solution, just don't see any better way.